client/src/routes/Session/OutroPortal.tsx

```diff
@@ -14,43 +14,121 @@ import VideoBase from './components/VideoBase/VideoBase';
 import usePreventGoingBack from '../../lib/navigation/hooks/usePreventGoingBack';
 import useNavigateWithFade from '../../lib/navigation/hooks/useNavigateWithFade';
 import Screen from '../../common/components/Screen/Screen';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import {useIsFocused} from '@react-navigation/native';
+import Button from '../../common/components/Buttons/Button';
+import {useTranslation} from 'react-i18next';
 
 const VideoStyled = styled(VideoBase)({
   ...StyleSheet.absoluteFillObject,
   flex: 1,
 });
 
+const reverseVideo = (url: string) => {
```

Comment:
need to do some research to see how robust this would be, but pretty neat that cloudnary has `e_reverse` option 🤓

Reply:
example:
```
cloudinary.video("ski_jump", {controls:true, transformation: [
    {overlay: "video:ski_jump", flags: "splice", effect: "reverse"},
    {overlay: "video:ski_jump", flags: "splice", effect: "accelerate:-50"},
    {effect: "brightness:10", radius: "max"}
    ]})
```

Reply:
It's something itchy with relying on the field being a cloudinary url when it basically could be anything. That's why I'm twisting and turning about this. But maybe we should go with just a regex to begin with.

Reply:
hmm, I see 🤔  

Are we planning or have indications that we're going to use something different than a cloudnary url on those fields at some point?

I could add a cloudinary check on the regexp as well if we go with that, and I right now if the regexp doesn't match it'll just return the unadulterated url... 

Could also add a hint on the CMS field asking that a cloudnary video asset is used 🤔

Reply:
Might be a good idea to add a sentry log when the regexp "fails" to find a match.

Reply:
[1ca8e87](https://github.com/29ki/29k/pull/962/commits/1ca8e87b5da3268dcf7dd8b0f7e90a6802104e2d)

Reply:
Nice I think it's good for now :)